### PR TITLE
update world val 2020 and 2021 ACS

### PR DESCRIPTION
**Pull request by libbymcclure**
updating the world val for the ACS metrics I've added 2020 and 2021 data to so far. looking at the world val for median age makes me question this current approach we're taking. thoughts?

### CODE REVIEW

File: config/data.js [hunk 1]
```diff
@@ -1209,6 +1226,17 @@ export default {
...
     'decimals': 0,
+    ,
+    'world_val': {
+      y_2010: 267583,
+      y_2015: 288817,
+      y_2016: 294618,
+      y_2017: 300865,
+      y_2018: 306457,
+      y_2019: 311848,
+      y_2020: 317185;
```
Comment (timstallmann) [inline config/data.js:1237]:
```suggestion
      y_2020: 317185,
```

@libbymcclure this is failing tests b/c of the semicolon should be a comma

**Comment by timstallmann** [comment]
World values for need to pull from census overall, currently they are aggregations of blockgroup/tract

**Review by JohnKilleen — APPROVED** [review]

File: config/data.js [hunk 2]
```diff
@@ -1209,6 +1226,17 @@ export default {
...
     'decimals': 0,
+    ,
+    'world_val': {
+      y_2010: 258578,
+      y_2015: 288817,
+      y_2016: 294618,
+      y_2017: 300865,
+      y_2018: 306457,
+      y_2019: 311848,
+      y_2020: 317665;
```
Comment (timstallmann) [inline config/data.js:1237]: @libbymcclure check this semicolon, it's why tests are failing still :/

Comment (libbymcclure) [reply config/data.js:1237]: ah sorry!

**Pull request merged**